scripts/osbuilder.sh

```diff
@@ -25,7 +25,7 @@ fi
 SCRIPT_NAME="${0##*/}"
 SCRIPT_DIR="$(dirname $(realpath -s $0))"
 REPO_URL=${REPO_URL:-https://download.clearlinux.org/current/x86_64/os/}
-
+EXTRA_PKGS=${EXTRA_PKGS:-""}
```

Comment:
nice!, could you add provide ENV VAR this to the docker command:
Something similar to:
https://github.com/clearcontainers/osbuilder/blob/master/Makefile#L24
https://github.com/clearcontainers/osbuilder/blob/master/Makefile#L41

Reply:
done